src/UniversalTelegramBot.cpp

```diff
@@ -685,7 +733,7 @@ String UniversalTelegramBot::sendPhoto(String chat_id, String photo,
   payload["chat_id"] = chat_id;
   payload["photo"] = photo;
 
-  if (caption)
+  if (!caption.isEmpty())
```

Comment:
Does this work if `null` is sent? I would have thought trying to call isEmpty of null would cause problems.

There probably is a bug in the original code though